layouts/shortcodes/beta-callout.html

```diff
@@ -0,0 +1,9 @@
+<div class="card beta-callout-card mb-4">
+    <div class="card-body d-flex flex-column">
+        <h5 class="card-title text-black mt-0 mb-1">Join the Beta!</h5>
+        <p class="card-text">{{.Inner}}</p>
+        <a href="{{.Get `url`}}" target="_blank" class="{{.Get `custom_class`}} btn btn-outline-primary pb-1 align-self-end d-flex flex-column justify-content-center" data-toggle="{{.Get `d_toggle`}}" data-target="{{.Get `d_target`}}">
```

Comment:
hey @StefonSimmons, this looks great.   can we render any non-required arguments conditionally here?   i.e. if `d_toggle` isn't provided in this example it will still output the attribute `data-toggle` with no value.

Reply:
hey @bgdeutsch I made those changes. thanks